test/refunder.ts

```diff
@@ -14,32 +14,14 @@ import {
 	FUNC_CALL_NOT_SUCCESSFUL
 } from './constants/error-messages.json';
 
-const ethToWei = (ether: string) => {
-	return ethers.utils.parseEther(ether);
-}
-
-const generateFuncIdAsBytes = (funcId: string) => {
-	funcId = ethers.utils.id(funcId);
-	return ethers.utils.arrayify(funcId.substr(0, 10));
-}
-
-const strToHex = (text: string) => {
-	let msg = '';
-	for (var i = 0; i < text.length; i++) {
-		var s = text.charCodeAt(i).toString(16);
-		while (s.length < 2) {
-		  s = '0' + s;
-		}
-		msg += s;
-	  }
-
-	return '0x' + msg;
-}
-
-const getXPercentFrom = (number: BigNumber, percent: number) => {
-	return number.div(BigNumber.from('100')).mul(BigNumber.from(percent.toString()));
-}
+import {
+	 ethToWei,
+	generateFuncIdAsBytes,
+	getXPercentFrom,
+	strToHex
+} from './utils/utils';
 
+const REFUNDER_VERSION = 1;
 
 describe("Refunder", function() {
```

Comment:
We need to add tests for:
- init function
- verify that on `updateRefundable` we are updating the registry as-well

Reply:
- init function >> is tested
- verify that on updateRefundable we are updating the registry as-well >> registry's tests cover that tests